sphinx/builders/singlehtml.py

```diff
@@ -62,7 +61,8 @@ def fix_refuris(self, tree: Node) -> None:
                 continue
             hashindex = refuri.find('#', hashindex + 1)
             if hashindex >= 0:
-                refnode['refuri'] = fname + refuri[hashindex:]
+                # all references are on the same page...
```

Comment:
Are you feeling down :( ?

```suggestion
                # all references are on the same page
```

Reply:
Hehe.  This is copied from an identical comment elsewhere in the code file :)  I'd prefer either to leave it the same, or update both.